ansible_collections/arista/avd/roles/eos_designs/docs/tables/fabric-settings.md

```diff
@@ -224,10 +224,10 @@
     # ```
     #
     # - By default, this will have no impact within the `eos_designs` role. Configs will still be generated by the `eos_cli_config_gen` role and will still be pushed by the `eos_config_deploy_eapi` directly to devices if used.
-    # - However, if the `eos_config_deploy_cvp` role is used to push configurations, CloudVision will ignore the devices flagged  as `is_deployed: false` and not attempt to configure them.
+    # - However, if the `cv_deploy` role is used to push configurations, CloudVision will ignore the devices flagged  as `is_deployed: false` and not attempt to configure them.
     # - If the device is not present in the network due to CloudVision not configuring the device, `eos_validate_state` role will fail tests on peers of the undeployed device trying to verify that interfaces are up.
     # - To overcome this and shutdown interfaces towards undeployed peers, the variable `shutdown_interfaces_towards_undeployed_peers` can be used, satisfying the `eos_validate_state` role interface tests.
```

Comment:
Need to check the rewrite against #5863 - or we can accept like this here and wait for #5863 to merge to update with better text

Reply:
> or we can accept like this here and wait for https://github.com/aristanetworks/avd/pull/5863 to merge to update with better text

That was my intend.